tests/test_core.py

```diff
@@ -7307,6 +7307,7 @@ def do_test_coroutine(self, additional_settings):
   def test_coroutine_asyncify(self):
     self.do_test_coroutine({'ASYNCIFY': 1})
 
+  @no_wasm_backend('EMTERPRETIFY causes JSOptimizer to run, which is disallowed')
```

Comment:
how about "unsupported" rather than "disallowed" ?

Reply:
or better yet, "Unsupported with the wasm backend" to make it more clear why

Reply:
Done. I copied the line from another test, so I fixed that too.